src/components/Results.jsx

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import { connect } from 'react-redux';
 
 const ResultsContainer = props => (
-  <div className='voting-result'>
-    <h4 className='voting-title'>What's better?</h4>
+  <div className='voting-result vr'>
```

Comment:
why don't use existed class?

Reply:
I was testing it out and forgot to remove it